extensions/hints/init.lua

```diff
@@ -46,6 +62,25 @@ local selectionCallback = nil
 local bumpThresh = 40^2
 local bumpMove = 80
 
+function split(str, pat)
```

Comment:
This function doesn't look to be called from anywhere. Either remove it or use it somewhere and make it `local`.